api/services/control/control.proto

```diff
@@ -157,3 +158,11 @@ message ListWorkersRequest {
 message ListWorkersResponse {
 	repeated moby.buildkit.v1.types.WorkerRecord record = 1;
```

Comment:
I think it would be good to add it to `WorkerRecord` as well in case in the future we will allow worker versions mismatch.

Reply:
^